File: Cryptool/app/src/main/java/io/github/nfdz/cryptool/CryptoolApp.java
package io.github.nfdz.cryptool;

import android.app.Application;
import android.support.v7.app.AppCompatDelegate;

import timber.log.Timber;

public class CryptoolApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        // Make sure we use vector drawables
        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);

        initLogger();
    }

    private void initLogger() {
        if (BuildConfig.DEBUG) {
            Timber.uprootAll();
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: src/CryptoolApp/app/src/main/java/io/github/nfdz/cryptool/CryptoolApp.kt
package io.github.nfdz.cryptool

import android.app.Application
import timber.log.Timber


class CryptoolApp : Application() {

    private val DB_NAME = "memotex.realm"
    private val SCHEMA_VERSION = 1L

    override fun onCreate() {
        super.onCreate()
        setupLogger()
    }

    private fun setupLogger() {
        if (BuildConfig.DEBUG) {
            Timber.uprootAll()
            Timber.plant(Timber.DebugTree())
        }
    }
}